packages/react-select/src/internal/ScrollCaptor.js

```diff
@@ -25,10 +25,6 @@ class ScrollCaptor extends Component<CaptorProps> {
     this.stopListening(this.scrollTarget);
   }
   startListening(el: HTMLElement) {
-    // bail early if no scroll available
-    if (!el) return;
```

Comment:
@pyreta I think this check could/should stay in here. Otherwise you'd undo #3419 I'm afraid :)

Reply:
ok fixed!